### PR TITLE
SPCTR-2964: Fix - Icon background and border hover color issue fix

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

Icon background and border hover color is not working as intended in both frontend and backend.
https://brainstormforce.atlassian.net/browse/SPCTR-2964

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: includes/blocks/icon/frontend.css.php [hunk 1]
```diff
@@ -83,15 +83,13 @@
 	),
 	UAGB_Block_Helper::uag_generate_border_css( $attr, 'icon' )
 );
-$selectors['.uagb-icon-wrapper .uagb-svg-wrapper:hover']     = array(
-	'border-color' => $attr['iconBorderHColor'],
-	'background'   => $hover_background,
-);
 
 // If using separate box shadow hover settings, then generate CSS for it.
 if ( $attr['useSeparateBoxShadows'] ) {
```
Comment (AbhijitNage123) [inline includes/blocks/icon/frontend.css.php:88]: If `useSeparateBoxShadows` is true then only border and background will apply.  @rajmohanreddy774 find out proper css selector

**Comment by rajmohanreddy774** [comment]
github issue so closing and will create new PR

File: includes/blocks/icon/frontend.css.php (hunk 1 above)
Comment (rajmohanreddy774) [reply includes/blocks/icon/frontend.css.php:92]: Because of using same selector we have to keep border and background inside if condition as well.

**Review by AbhijitNage123 — APPROVED** [review]

File: includes/blocks/icon/frontend.css.php (hunk 1 above)
Comment (AbhijitNage123) [reply includes/blocks/icon/frontend.css.php:92]: I can see On line No 157 we have manage the case. So marking it as resolve.

File: README.md [hunk 2]
```diff
@@ -176,6 +176,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 * Fix: Testimonial - Resolved equal height issue in frontend when 2 columns per slide is set.
 * Fix: Buttons - Resolved an issue in which the space between the icon and text would not apply in RTL sites.
 * Fix: Image - Resolved an issue in which the tablet and mobile border css would not be applied as intended.
+* Fix: Icon - Resolved issues in which icon background and border hover color would not work as intended.
```
Comment (bsf-zanev) [inline README.md:179]:
```suggestion
* Fix: Icon - Resolved an issue in which the icon's background and border color on hover would not work as intended. 
```

**Review by bsf-zanev — APPROVED** [review]
Looks good, just a changelog update needed.

File: README.md (hunk 2 above)
Comment (rajmohanreddy774) [reply README.md]: updated.

**Review by adi3890 — APPROVED** [review]

**Pull request merged**